interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -826,6 +826,26 @@ public void unimplementedMethod() {
     }
   }
 
+  /** Start a fullDuplexCall to an unresponsive server and verify the deadline expires. */
+  @Test(timeout = 10000)
+  public void timeoutOnSleepingServer() {
+    TestServiceGrpc.TestService stub = TestServiceGrpc.newStub(channel)
+        .withDeadlineAfter(1, TimeUnit.MILLISECONDS);
+    @SuppressWarnings("unchecked")
+    StreamObserver<StreamingOutputCallResponse> responseObserver = mock(StreamObserver.class);
+    StreamObserver<StreamingOutputCallRequest> requestObserver
+        = stub.fullDuplexCall(responseObserver);
+    requestObserver.onNext(StreamingOutputCallRequest.newBuilder()
+        .setPayload(Payload.newBuilder()
+            .setBody(ByteString.copyFrom(new byte[27182])))
+        .build());
+
+    ArgumentCaptor<Throwable> captor = ArgumentCaptor.forClass(Throwable.class);
+    verify(responseObserver, timeout(OPERATION_TIMEOUT)).onError(captor.capture());
+    assertEquals(Status.DEADLINE_EXCEEDED, Status.fromThrowable(captor.getValue()));
```

Comment:
Since the equality of our Status is not very clear, I'd suggest comparing the code.